README.adoc

```diff
@@ -83,6 +83,7 @@ image:docs/images/file_parser.png[notebook]
 
 ==== Build Notebook
 
+* Name Machine Learning task name (Eg: Training)
```

Comment:
Had to read it a couple times due to the double name in the sentence. Maybe "Choose a name for the Machine Leaning task folder". The part, "task folder", then repeats below when you say "will be saved in the workspace under the task folder name". WDYT? Not a blocker :+1: but I think it can be merged after #38 is merged too?

Reply:
will change it